catalog/log-export/folder/logbucket-export/export.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# API Activation
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  name: my-project-id-logbucket # kpt-set: ${project-id}-logbucket
+  namespace: my-namespace # kpt-set: ${namespace}
+  annotations:
+    cnrm.cloud.google.com/blueprint: cnrm/landing-zone:log-export/v0.4.0
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+    cnrm.cloud.google.com/disable-dependent-services: "false"
+    cnrm.cloud.google.com/project-id: my-project-id # kpt-set: ${project-id}
+spec:
+  resourceID: logging.googleapis.com
+---
+# Org Sink to Log Bucket
+apiVersion: logging.cnrm.cloud.google.com/v1beta1
+kind: LoggingLogSink
+metadata:
+  name: 123456789012-fldrlogbucketsink # kpt-set: ${folder-id}-logbucketsink
+  namespace: my-namespace # kpt-set: ${namespace}
+  annotations:
+    cnrm.cloud.google.com/blueprint: cnrm/landing-zone:log-export/v0.4.0
+spec:
+  folderRef:
+    external: "123456789012" # kpt-set: ${folder-id}
```

Comment:
I think what @wenxuan-xia meant is we should align with the other blueprints and make this a reference rather than using an external. 
https://github.com/GoogleCloudPlatform/blueprints/blob/a814f19df7a68e9e099203468177b4921bbb102b/catalog/log-export/folder/storage-export/export.yaml#L41-L42
https://github.com/GoogleCloudPlatform/blueprints/blob/a814f19df7a68e9e099203468177b4921bbb102b/catalog/log-export/folder/pubsub-export/export.yaml#L41-L42
https://github.com/GoogleCloudPlatform/blueprints/blob/a814f19df7a68e9e099203468177b4921bbb102b/catalog/log-export/folder/bigquery-export/export.yaml#L41-L42